module/programs/helper.js

```diff
@@ -73,7 +74,7 @@ module.exports = class ProgramsHelper {
    * @returns {JSON} - create program.
    */
 
-  static create(data) {
+  static create(data, checkDate = false) {
```

Comment:
@borkarsaish65 Better to update function signature as well

Reply:
done